diff --git i/build/Settings.cs w/build/Settings.cs
index 0ccd4c5..28d78cd 100644
--- i/build/Settings.cs
+++ w/build/Settings.cs
@@ -91,6 +91,14 @@ namespace Build
 
         public static List<BuildConfiguration> LinuxBuildConfigurations = new List<BuildConfiguration>()
         {
+            new BuildConfiguration()
+            {
+                ConfigId = ConfigId.any_any,
+                SourceProjectFileName = "extensions.csproj",
+                RuntimeIdentifier = "any",
+                PublishReadyToRun = false,
+                PublishBinDirectorySubPath = "bin"
+            },
             new BuildConfiguration()
             {
                 ConfigId = ConfigId.linux_x64,
