diff --git a/docker/release/config/opensearch/opensearch-docker-entrypoint-3.x.sh b/docker/release/config/opensearch/opensearch-docker-entrypoint-3.x.sh
index d998663..39a7397 100755
--- a/docker/release/config/opensearch/opensearch-docker-entrypoint-3.x.sh
+++ b/docker/release/config/opensearch/opensearch-docker-entrypoint-3.x.sh
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+# Reexec as the opensearch user if running as root.
+if [ "$(id -u)" -eq 0 ]; then
+	echo "ENTRYPOINT: Dropping root..." >&2
+	exec gosu opensearch docker-entrypoint.sh "$@"
+fi
+
 # Copyright OpenSearch Contributors
 # SPDX-License-Identifier: Apache-2.0
 
