diff --git a/build.gradle b/build.gradle
index af480ed..7a0fd97 100644
--- a/build.gradle
+++ b/build.gradle
@@ -487,6 +487,7 @@ configurations {
             force "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:${versions.jackson}"
             force "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
             force "io.netty:netty-buffer:${versions.netty}"
+            force "io.netty:netty-codec-http:${versions.netty}"
             force "io.netty:netty-common:${versions.netty}"
             force "io.netty:netty-handler:${versions.netty}"
             force "io.netty:netty-transport:${versions.netty}"
