diff --git a/src/dev/build/lib/fs.ts b/src/dev/build/lib/fs.ts
index d353a1957a..ce9c1ce8df 100644
--- a/src/dev/build/lib/fs.ts
+++ b/src/dev/build/lib/fs.ts
@@ -42,7 +42,7 @@ import vfs from 'vinyl-fs';
 import File from 'vinyl';
 import del from 'del';
 import deleteEmpty from 'delete-empty';
-import tar, { ExtractOptions } from 'tar';
+import { extract as tarExtract, type ExtractOptions } from 'tar';
 import { ToolingLog } from '@osd/dev-utils';
 import { standardize } from '@osd/cross-platform';
 
@@ -287,7 +287,7 @@ export async function untar(
   await pipelineAsync(
     fs.createReadStream(source),
     createGunzip(),
-    tar.extract({
+    tarExtract({
       ...extractOptions,
       cwd: destination,
     })
diff --git a/src/plugins/data_source_management/public/components/edit_data_source/edit_data_source.tsx b/src/plugins/data_source_management/public/components/edit_data_source/edit_data_source.tsx
index f878090791..c4ff4a6436 100644
--- a/src/plugins/data_source_management/public/components/edit_data_source/edit_data_source.tsx
+++ b/src/plugins/data_source_management/public/components/edit_data_source/edit_data_source.tsx
@@ -9,8 +9,6 @@ import { useEffectOnce } from 'react-use';
 import { EuiSpacer } from '@elastic/eui';
 import { i18n } from '@osd/i18n';
 import { FormattedMessage } from '@osd/i18n/react';
-// @ts-expect-error TS6133 TODO(ts-error): fixme
-import { c } from 'tar';
 import { UiSettingScope } from '../../../../../core/public';
 import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
 import {
